src/controllers/userController.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import { User } from '../models/user';
+import { UserRepository } from '../repository/userRepository';
```

Comment:
Please, keep the standard using the plural to the folders, `models`, `controllers`, `repositories`, etc.